bin/build

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+TAG=latest
+
+MIQ_REF=${MIQ_REF:-master}
+SUI_REF=${SUI_REF:-master}
+APPLIANCE_REF=${APPLIANCE_REF:-master}
+
+MIQ_ORG=${MIQ_ORG:-ManageIQ}
+SUI_ORG=${SUI_ORG:-ManageIQ}
+APPLIANCE_ORG=${APPLIANCE_ORG:-ManageIQ}
+
+while getopts "t:d:r:nh" opt; do
+  case $opt in
+    d) IMAGE_DIR=$OPTARG ;;
+    n) NO_CACHE="true" ;;
+    p) PUSH="true" ;;
+    r) REPO=$OPTARG ;;
+    t) TAG=$OPTARG ;;
+    h) echo "Usage: $0 -d IMAGE_DIR -r IMAGE_REPOSITORY [-hnp] [ -t IMAGE_TAG ]"; exit 1
+  esac
+done
+
+if [ -z "$IMAGE_DIR" ]; then
+  echo "Required parameter for image directory (-d) is missing"
+  exit 1
+fi
+
+if [ -z "$REPO" ]; then
+  echo "Required parameter for repository (-r) is missing"
+  exit 1
+fi
+
+images="manageiq-base-worker manageiq-orchestrator manageiq-webserver-worker manageiq-ui-worker"
+
+set -e
+
+pushd $IMAGE_DIR
+  if [ -n "$NO_CACHE" ]; then
+    docker build -t $REPO/manageiq-base:$TAG --no-cache \
+      --build-arg MIQ_REF=$MIQ_REF \
+      --build-arg SUI_REF=$SUI_REF \
+      --build-arg APPLIANCE_REF=$APPLIANCE_REF \
+      --build-arg MIQ_ORG=$MIQ_ORG \
+      --build-arg SUI_ORG=$SUI_ORG \
+      --build-arg APPLIANCE_ORG=$APPLIANCE_ORG \
+      manageiq-base
+  else
+    docker build -t $REPO/manageiq-base:$TAG \
+      --build-arg MIQ_REF=$MIQ_REF \
+      --build-arg SUI_REF=$SUI_REF \
+      --build-arg APPLIANCE_REF=$APPLIANCE_REF \
+      --build-arg MIQ_ORG=$MIQ_ORG \
+      --build-arg SUI_ORG=$SUI_ORG \
+      --build-arg APPLIANCE_ORG=$APPLIANCE_ORG \
+      manageiq-base
+  fi
+
+  for image in $images; do
+    docker build -t $REPO/$image:$TAG --build-arg FROM_REPO=$REPO --build-arg FROM_TAG=$TAG $image
+  done
+popd
+
+if [ -n "$PUSH" ]; then
+  for image in images; do
```

Comment:
We need to push `manageiq-base` as well.

Reply:
Do we? Why? We don't run it.

Reply:
But the next image depends on this, so if we don't push the latest, people building dependent images won't have the latest if not building everything using this script.